aspnetcore/signalr/redis-backplane.md

```diff
@@ -212,9 +212,16 @@ services.AddSignalR()
 
 :::moniker-end
 
-## Redis Clustering
+## Redis Cluster
+
+[Redis Cluster](https://redis.io/topics/cluster-spec) utilizes multiple simultaneously active Redis servers to achieve high availability. When Redis Cluster is used as the backplane for SignalR, messages are delivered to all of the nodes of the cluster without code modifications to the app.
+
+There's a tradeoff between the number of nodes in the cluster and the availability of the backplane, increasing one decreases the other. When additional nodes are added to the cluster, the backplane's availability to process messages is reduced.
```

Comment:
We lost the throughput messaging now 😆

Reply:
It seems like the main concept concern for the second paragraph is "throughput" because increasing nodes = increasing availability, which is good. The throughput of the backplane decreases when nodes are added because all nodes receive messages.

I'll try again with that 2nd paragraph 😈.